rdtho2525/whats-cookin
# Feature/user stories iteration2

## Is this a fix or a feature? ⬇️

Feature

## What is the change? ⬇️

This allows a user to search their favorite recipes with the same filters that can be used on all recipes. Also, layout and styling were adjusted in the navigation pane and the hero image area.

## Where should the reviewer start? ⬇️

All function expressions on lines 131-195 in scripts.js are new or refactored. Additionally, the navigation area of index.html has been refactored to allow for increased flexibility in styling

## How should this be tested? ⬇️

Open index.html in a browser. All cards should be populated on load. Using relevant search values, search through each array with each filter. Additionally, test that irrelevant search values do not produce any results. After each result, the header above the cards view should reflect what you are viewing (including array and search term, if applicable.
